### PR TITLE
Issue #1596: Removed dependence on locale-specific DecimalFormatSymbols.

### LINKED ISSUE #1596: Pattern finder finds different patterns on different systems
Depending on what system (e.g. your computer, another computer, Travis CI) you run a job with a `Pattern finder` analyzer in it, its results may in practice vary.

E.g. when I run the sample `Customer age analysis` job, which is provided with DataCleaner, I run the job and I inspect the results of the Birthdate patterns, on my computer it finds three patterns: `####-##-##`, `####.##.##` and `a/a`. On someone else's computer it finds  `####-##-##`, `##########` and `a/a` instead.

I'm under the impression that this may be caused by the fact the the Pattern Finder uses the java.text.DecimalFormatSymbols class and instantiates it without explicitly passing a locale, as a result of which the default locale of the JVM is used, which may differ per system.

### PR DESCRIPTION
**Pull request by kaspersorensen**
Fixes #1596

### CODE REVIEW

File: components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/TokenizerConfiguration.java [hunk 1]
```diff
@@ -29,7 +28,7 @@
 public class TokenizerConfiguration implements Serializable {
 
     private static final long serialVersionUID = 1L;
-
+
```
Comment (arjansh) [inline components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/TokenizerConfiguration.java]: Why adding the whitespaces here?

File: components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/PatternFinderAnalyzer.java [hunk 2]
```diff
@@ -118,13 +118,14 @@
     String[] predefinedTokenPatterns;
 
     @Configured(required = false, order = 12)
-    Character decimalSeparator = DecimalFormatSymbols.getInstance().getDecimalSeparator();
+    Character decimalSeparator = '.';
```
Comment (arjansh) [inline components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/PatternFinderAnalyzer.java]: Isn't it better to store the '.', ',' and '-' in constants, taking into account that you reuse them in the `TokenizerConfiguration` class

File: components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/TokenizerConfiguration.java (hunk 1 above)
Comment (kaspersorensen) [reply components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/TokenizerConfiguration.java]: By mistake

File: components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/PatternFinderAnalyzer.java (hunk 2 above)
Comment (kaspersorensen) [reply components/pattern-finder/src/main/java/org/datacleaner/beans/stringpattern/PatternFinderAnalyzer.java]: Added constants in TokenConfiguration

**Comment by arjansh** [comment]
LGTM!

**Pull request merged**